docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -59,13 +65,11 @@ cluster.initial_master_nodes:
   - master-c
 --------------------------------------------------
 
-Like all node settings, it is also possible to specify the initial set of master
-nodes on the command-line that is used to start Elasticsearch:
```

Comment:
We've removed most mention of this from elsewhere in the docs, I suspect because it only works for certain installation methods. No need to say it here either.